repo/mpi-pingpong/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Lawrence Livermore National Security, LLC and other
+# Benchpark Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from spack.package import *
+
+
+class MpiPingpong(CMakePackage):
+
+    git = "/usr/WS1/mckinsey/benchpark/mpi-pingpong/"
```

Comment:
how to set this path dynamically? Can we do it like ramble, e.g. `{local_path}`

Reply:
you should use #620 for this.